SharedCode/Repository/PokemonRepository.cs

```diff
@@ -12,6 +16,8 @@ namespace SharedCode.Repository
 {
 	public class PokemonRepository : IPokemonRepository
 	{
+        IDatabaseManager dataBaseManager = IocContainer.GetDependency<DatabaseManager>();
```

Comment:
Why are you declaring this if it looks like you're not using this in this file

Reply:
Removed unused variable.